python/lsst/daf/butler/configs/old_dimensions/README.md

```diff
@@ -0,0 +1,4 @@
+These files are used by
+[daf_butler_migrate](https://github.com/lsst-dm/daf_butler_migrate) to verify
+and update the dimension universe stored in the database when upgrading the
+database schema.
```

Comment:
Maybe add a note here that the naming convention is `<namespace>_universe<version>.yaml` ?